components/transaction-stats.tsx

```diff
@@ -4,14 +4,18 @@ import {TransactionStatus} from '@interfaces/enums/transaction-status';
 export default function TransactionStats({status = null}: {status: TransactionStatus}) {
   const ColorMap = {
     [TransactionStatus.pending]: `warning`,
-    [TransactionStatus.processing]: `info`,
+    [TransactionStatus.processing]: `blue-info`,
```

Comment:
info is a bootstrap prefix, if we don't have the correct color for info then we should be editing _the variable of that colour_ and not adding a new info

also, the pending state is repeated